<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qGLRegisterAccount</class>
 <widget class="QDialog" name="qGLRegisterAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetNewAna" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>844</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>0</y>
      <width>811</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>注册用户</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>844</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 119, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>130</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>机构:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editOrganizations">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDepartment">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>180</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>科室:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>280</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>性别:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>230</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>180</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPhone">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>手机:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="editUser">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>101</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSex">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <zorder>lblBg</zorder>
   <zorder>lblTitle</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>editOrganizations</zorder>
   <zorder>editDepartment</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>editName</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>editPhone</zorder>
   <zorder>label_6</zorder>
   <zorder>editPassword</zorder>
   <zorder>editUser</zorder>
   <zorder>label_7</zorder>
   <zorder>editPassword2</zorder>
   <zorder>label_8</zorder>
   <zorder>cmbSex</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
